.eslintrc.cjs

```diff
@@ -86,6 +86,19 @@ module.exports = {
           }
         ],
 
+        // Check relative import paths use aliases
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: [
+              {
+                group: ['./', '../'],
+                message: 'Relative imports are not allowed.'
```

Comment:
Sorry, I forgot this message also said relative imports not aliases

Unless the default is obvious enough and we don't need one?

Thanks @KiranVarma02

```suggestion
                message: "Please use '~/*' import alias instead.'
```

Reply:
perfect